### ISSUE TITLE
Crashes relating to KeyboardCompatibilityView.

### ISSUE THREAD

**Issue by bawolf** [GetStream/stream-chat-react-native#199]
We're seeing crashes in our crashlytics like:
```bash
 Fatal Exception: RCTFatalException: Unhandled JS Exception: TypeError: null is not an object (evaluating 'n.rootChannelView.measureInWindow')
```
It seems to be caused by component:
https://github.com/GetStream/stream-chat-react-native/blob/1eec0783cfb5400d6874213aeb009e61dcdbdc4e/src/components/KeyboardCompatibleView.js#L122

I haven't had any luck making a sample app to reproduce yet, but it seems to happen most when a few people are typing back and forth quickly.

**Comment by vishalnarkhede** [comment]
Thanks for bringing this up @bawolf . I will look into it and get back to you :)

**Comment by vishalnarkhede** [comment]
Hey @bawolf fix has been published on `0.10.2`. Please reopen the issue if you still see the error :)
